tests/shieldhit/test_shieldhit_converter.py

```diff
@@ -0,0 +1,48 @@
+import filecmp
+import logging
+from pathlib import Path
+
+import pytest
+
+from converter.api import run_parser
+from converter.shieldhit.parser import Parser
+
+
+def test_parser(sh12a_parser : Parser) -> None:
+    """Check if parser is created correctly"""
+    assert sh12a_parser
+    assert sh12a_parser.info['version'] == ''
+    assert sh12a_parser.info['simulator'] == 'shieldhit'
```

Comment:
There is no check for `label` value.

```suggestion
    assert sh12a_parser.info['simulator'] == 'shieldhit'
    assert sh12a_parser.info['label'] == ''
```